Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -188,6 +188,8 @@ private async Task<bool> CanReadOneAsync(string resource)
                 }
             case Resource.NEWS:
                 return true;
+            case Resource.STUDENT_SCHEDULE:
+                return user_groups.Contains(AuthGroup.Advisors);
```

Comment:
Would it be better to use `ScheduleService.CanReadStudentSchedules` for this authorization check? That way, if the logic ever becomes more complicated in the future, the real authorization and the front end check are both controlled from the same function.